reflex-dom/java/org/reflexfrp/reflexdom/MainWidget.java

```diff
@@ -36,7 +41,27 @@ private static Object startMainWidget(Activity a, String url, long jsaddleCallba
         public void onPageFinished(WebView _view, String _url) {
           wv.evaluateJavascript(initialJS, null);
         }
-      });
+    });
+
+    wv.setWebChromeClient(new WebChromeClient() {
+        // Need to accept permissions to use the camera and audio
+        @Override
+        public void onPermissionRequest(final PermissionRequest request) {
+            a.runOnUiThread(new Runnable() {
+                @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
Honestly I don't exactly know what that @TargetApi stuff does. I haven't done any Android development before and the Internet is only partly helpful. It is there, because I found it that way on the Internet and I guess there is a good reason.